### ISSUE TITLE
Microphone not working at all

### ISSUE THREAD

**Issue by faedy2** [pop-os/pop#1274]
**Distribution (run `cat /etc/os-release`):**
 NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
N/A

**Issue/Bug Description:**
My microphone used to not work, so I would have to plug in a pair of headphones to the aux port, then unplug them, and then the mike would work. This bizarre solution no longer works. Please note that my mike works fine on Windows.

**Steps to reproduce (if you know):**
Attempt to use microphone on any application. See if the computer detects any input from the microphone.

**Expected behavior:**
Microphone should pick up sound in any application that supports it.

**Other Notes:**
If anyone has a quick-fix solution, I would appreciate it. Thanks.

**Comment by jacobgkau** [comment]
What hardware are you using? What audio card/motherboard?

What do you see under the "Input Devices" and "Configuration" tabs of pavucontrol? https://support.system76.com/articles/audio/ 

---

System76 customers can reach out to [support](https://support.system76.com/) for technical assistance. For non-System76 hardware, you can seek community support on [Reddit](https://reddit.com/r/pop_os/) or [Mattermost](https://chat.pop-os.org).

**Comment by faedy2** [comment]
Realtek ALC295 is the audio card
Under "Input Devices", I have Internal Microphone and Headset Microphone (unplugged) under "Built-in Audio Analog Stereo"
Under "Configuration", I have GP106 High Definition Audio Controller set to 'Digital Stereo (HDMI) Output' and Built-in Audio set to 'Analog Stereo Duplex'
One thing I noticed today was that I was able to speak into my plugged in headset just fine once, and it was detected as 'Internal Microphone' instead of as 'Headset Microphone'. Once I unplugged the headset, the laptop microphone continued to not work, and plugging the headset back in stopped the headset microphone from working either.
So long story short, I can sometimes get the headset microphone to work only once, and once it is unplugged, I may need to either restart my audio driver/the laptop to get a working microphone again.

**Comment by Ashc0d** [comment]
I also have the same problem.
To get my  microphone working again i have to shutdown the pc and boot into pop-os then the problem will be  solved
temporarily.

**Comment by faedy2** [comment]
Microphone works on boot now after updating to 20.10, but there is still a ton of random distortion.

**Comment by ShashankArora92** [comment]
My microphone is also not working in Pop os 20.04. I tried with Bluetooth earphone's mic still not recording any sound.
Please help.

**Comment by Ashc0d** [comment]
@ShashankArora92 
Check whether the system has selected it input source or not
check in the Settings>sound>Input

**Comment by ShashankArora92** [comment]
I checked all those, yeah they were the same but did not work for me
.

**Comment by jack75015** [comment]
Hello
I got exactly the same problem since this morning..
@ShashankArora92 did you fix it? :)

**Comment by ShashankArora92** [comment]
@jack75015 brother I attached external mic and found out that the problem is not with the software, it is with my internal microphone. It need to be replaced.

**Comment by thelastindian** [comment]
Please try this https://www.reddit.com/r/pop_os/comments/vu6p2i/microphone_not_working_on_my_laptop/ 

Solution: Installed pavucontrol and increased input devices volume slightly above silent base https://imgur.com/a/KdgEPr6

I hope this works out for you.

**Comment by cmarincia** [comment]
I have the same problem, I cannot use the internal microphone since it is marked as unavailable on my laptop.
